config/settings/clever_cloud.py

```diff
@@ -0,0 +1,51 @@
+from .base import *  # noqa
+
+DEBUG = os.environ.get("DJANGO_DEBUG", True)
+
+ALLOWED_HOSTS = ["127.0.0.1", ".cleverapps.io"]
+
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+
+# Django-extensions.
+# ------------------------------------------------------------------------------
+
+INSTALLED_APPS += ["django_extensions"]  # noqa F405
+
+# Django-debug-toolbar.
+# ------------------------------------------------------------------------------
+
+INSTALLED_APPS += ["debug_toolbar"]  # noqa F405
+
+INTERNAL_IPS = ["127.0.0.1"]
+
+# Enable django-debug-toolbar with Docker.
+import socket
+
+_, _, ips = socket.gethostbyname_ex(socket.gethostname())
+INTERNAL_IPS += [ip[:-1] + "1" for ip in ips]
```

Comment:
Je veux bien un petit commentaire, je suis pas sûr de bien comprendre pourquoi on doit ajouter un `1` en fin d'IP.

Reply:
Ah, ça, ce n'est pas de moi ! :) J'ai simplement copié / collé le fichier de conf dev.py et je n'ai pas essayé de modifier cette partie.

Reply:
Je reste aussi curieux du pourquoi

Reply:
https://github.com/pydanny/cookiecutter-django/pull/1520/files

Reply:
Bref une sombre histoire de Docker, il doit y avoir une explication quelque part dans ce thread mais j'ai la flemme de creuser https://stackoverflow.com/questions/26898597/django-debug-toolbar-and-docker